### PR TITLE
User and Role entities, services and repositories was added

### PR DESCRIPTION
**Pull request by Mandybur800**

### CODE REVIEW

File: src/main/java/ua/bank/bankservice/controllers/DataInject.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    private final RoleService roleService;
+    private final UserService userService;
+
+    @PostConstruct
+    private void init() {
+        Role user = new Role();
+        user.setRoleName(Role.Roles.USER);
+        roleService.create(user);
+        Role admin = new Role();
+        admin.setRoleName(Role.Roles.ADMIN);
```
Comment (IvanLuchkin) [inline src/main/java/ua/bank/bankservice/controllers/DataInject.java]: Such enum naming leads to confusion, consider refactoring to RoleType or smth else

File: src/main/java/ua/bank/bankservice/controllers/UserController.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+
+    @GetMapping("/{userId}")
+    public User getById(@PathVariable Long userId) {
+        return userService.get(userId).orElseThrow(() ->
+                new RuntimeException("Can't get user with id:" + userId));
+    }
+}
```
Comment (IvanLuchkin) [inline src/main/java/ua/bank/bankservice/controllers/UserController.java]: I don't think that we need to create any controllers at this point

**Review by IvanLuchkin — CHANGES_REQUESTED** [review]

File: src/main/java/ua/bank/bankservice/controllers/UserController.java (hunk 2 above)
Comment (Mandybur800) [reply src/main/java/ua/bank/bankservice/controllers/UserController.java:22]: I add this controller only to check the correct work of this app.

**Review by IvanLuchkin — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -6,3 +6,6 @@ script:
 cache:
   directories:
     - $HOME/.m2
+before_install:
+  - chmod +x mvnw
+  -
```
Comment (boroda4436) [inline .travis.yml]: Do we need this `-` symbol at the end of the file?

File: src/main/java/ua/bank/bankservice/service/impl/RoleServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+    @Override
+    public Role create(Role role) {
+        return roleRepository.save(role);
+    }
+
+    @Override
+    public Role getByName(String name) {
+        return roleRepository.getByName(Role.RoleType.valueOf(name))
+                .orElseThrow(() -> new RuntimeException("Can't get role '" + name + "' from db"));
```
Comment (boroda4436) [inline src/main/java/ua/bank/bankservice/service/impl/RoleServiceImpl.java]: You can create your own exception. Smth like `EntityNotFoundException`. In the future, you can catch it in GlobalExceptionHandler

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**